digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -85,11 +85,11 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService, implici
     }
 
     def extractLinkId(massTransitStop: PersistedMassTransitStop): (String, Option[Long]) = {
-      "link_id" -> Some(massTransitStop.linkId)
+      if(Some(massTransitStop.linkId).contains(0)) "link_id" -> None else "link_id" -> Some(massTransitStop.linkId)
     }
 
     def extractMvalue(massTransitStop: PersistedMassTransitStop): (String, Option[Double]) = {
-      "m_value" -> Some(massTransitStop.mValue)
+      if(Some(massTransitStop.mValue).contains(0)) "m_value" -> None else "m_value" -> Some(massTransitStop.mValue)
```

Comment:
Same as above:
```scala
    def extractMvalue(massTransitStop: PersistedMassTransitStop): (String, Option[Double]) = {
      "m_value" -> (if(massTransitStop.mValue == 0) None else Some(massTransitStop.mValue))
    }

Reply:
BTW, in this case, the mvalue can be 0 if we want to create the asset at the beginning of the road link.
Does this still make sense with this info?

Reply:
Same idea as previous comment
    
```scala
def extractMvalue(massTransitStop: PersistedMassTransitStop, isMassServicePoint: Boolean): (String, Option[Long]) = {
      "m_value" -> (if(isMassServicePoint) None else Some(massTransitStop.mValue))
    }
```

Reply:
@marquesfm Check modification on this code.
This part was rollbacked and added a if on the Map construction